ivadomed/maths.py

```diff
@@ -0,0 +1,40 @@
+import nibabel as nib
```

Comment:
Could you please add this file to the API / RST? thanks :)

Reply:
Will do

Reply:
done in 064412a

Reply:
All good to me now :-)